src/Components/SearchBar.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+var text = "practise";
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div className="SearchBar">
+        <header className="app__header">
+          <section class="search">
+            <form action="">
+              <input
+                class="text"
+                placeholder="Type in a city name..."
+                name="search"
+              />
+              <button class="btn">{this.props.buttonTitle}</button>
```

Comment:
While we practiced using props by specifying the button title, it's probably unnecessary to do this in the final version.